src/java/org/apache/cassandra/service/accord/AccordService.java

```diff
@@ -350,6 +374,7 @@ private AccordService(Id localId)
         Invariants.checkState(localId != null, "static localId must be set before instantiating AccordService");
         logger.info("Starting accord with nodeId {}", localId);
         AccordAgent agent = FBUtilities.construct(CassandraRelevantProperties.ACCORD_AGENT_CLASS.getString(AccordAgent.class.getName()), "AccordAgent");
+        agent.setNodeId(localId);
```

Comment:
this was added for testing.  `AccordTopologyMixupTest` defines its own `AccordAgent` so it can provide debug details for timed out txns